data/mods/Aftershock/maps/furniture_and_terrain/terrain_spaceship.json

```diff
@@ -22,6 +22,519 @@
+        { "item": "scrap", "count": [ 1, 5 ] }
+      ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_afs_gun_ship_engine",
+    "name": "Iblis Interplanetary Drive",
+    "looks_like": "t_plut_generator",
+    "description": "The housing and guts of an Interplanetary drive engine.  It is plastered with warning signs.  If it's not working you could probably try to break into it for parts.",
+    "symbol": "0",
+    "color": "green_red",
+    "move_cost": 0,
+    "coverage": 80,
+    "flags": [ "NOITEM", "SEALED", "REDUCE_SCENT", "PERMEABLE" ],
+    "bash": {
+      "str_min": 70,
+      "str_max": 400,
+      "explosive": 100,
+      "sound": "metal screeching!",
+      "ter_set": "t_metal_floor",
+      "sound_fail": "clang!",
+      "items": [
+        { "item": "afs_circuitry_1", "count": [ 4, 16 ] },
+        { "item": "afs_circuitry_2", "count": [ 1, 6 ] },
+        { "item": "afs_circuitry_3", "count": [ 0, 1 ] },
+        { "item": "afs_circuitry_3", "count": [ 0, 1 ] },
+        { "item": "afs_heat_2", "count": [ 0, 1 ] },
+        { "item": "afs_energy_storage_2", "count": [ 1, 6 ] },
+        { "item": "afs_energy_storage_4", "count": [ 0, 1 ] },
+        { "item": "afs_material_1", "count": [ 12, 18 ] }
+      ]
+    }
+  },
+  {
+    "type": "terrain",
+    "id": "t_afs_gun_ship_engine_console",
+    "name": "Iblis Interplanetary Drive Console",
+    "looks_like": "t_console",
+    "description": "The control panel for the engines and generators that power the drive.  It would likely render the ship irrecoverable but it might do some damage to any robots near or in the ship if you set the engines to overload.",
+    "symbol": "0",
+    "color": "green_red",
+    "move_cost": 0,
+    "coverage": 80,
+    "examine_action": {
+      "type": "effect_on_condition",
+      "effect_on_conditions": [
+        {
+          "id": "EOC_AFS_GUNSHIP_DRIVE_OVERLOAD",
+          "condition": { "u_query": "Do you want to overload the ship's drive?", "default": true },
+          "effect": [
+            { "u_message": "You hear a high pitched whine start and increase in volume.", "popup": true },
+            { "u_cast_spell": { "id": "gunship_drive_overload" } }
+          ]
+        }
+      ]
+    },
+    "flags": [ "NOITEM", "SEALED", "REDUCE_SCENT", "PERMEABLE" ],
+    "bash": {
+      "str_min": 70,
+      "str_max": 400,
+      "sound": "metal screeching!",
+      "ter_set": "t_metal_floor",
+      "sound_fail": "clang!",
+      "items": [
+        { "item": "afs_circuitry_1", "count": [ 4, 16 ] },
+        { "item": "afs_circuitry_2", "count": [ 1, 6 ] },
+        { "item": "afs_circuitry_3", "count": [ 0, 1 ] },
+        { "item": "afs_circuitry_3", "count": [ 0, 1 ] },
+        { "item": "afs_heat_2", "count": [ 0, 1 ] },
+        { "item": "afs_energy_storage_2", "count": [ 1, 6 ] },
+        { "item": "afs_energy_storage_4", "count": [ 0, 1 ] },
+        { "item": "afs_material_1", "count": [ 12, 18 ] }
+      ]
+    }
+  },
```

Comment:
```suggestion
  {
    "type": "terrain",
    "id": "t_afs_gun_ship_engine_console",
    "name": "Iblis Interplanetary Drive Console",
    "looks_like": "t_console",
    "description": "The control panel for the engines and generators that power the drive.  It would likely render the ship irrecoverable but it might do some damage to any robots near or in the ship if you set the engines to overload.",
    "symbol": "0",
    "color": "green_red",
    "move_cost": 0,
    "coverage": 80,
    "examine_action": {
      "type": "effect_on_condition",
      "effect_on_conditions": [
        {
          "id": "EOC_AFS_GUNSHIP_DRIVE_OVERLOAD",
          "condition": { "u_query": "Do you want to overload the ship's drive?", "default": true },
          "effect": [
            { "u_message": "You hear a high pitched whine start and increase in volume.", "popup": true },
            { "u_cast_spell": { "id": "gunship_drive_overload" } }
          ]
        }
      ]
    },
    "flags": [ "NOITEM", "SEALED", "REDUCE_SCENT", "PERMEABLE" ],
    "bash": {
      "str_min": 70,
      "str_max": 400,
      "sound": "metal screeching!",
      "ter_set": "t_metal_floor",
      "sound_fail": "clang!",
      "items": [
        { "item": "afs_circuitry_1", "count": [ 4, 16 ] },
        { "item": "afs_circuitry_2", "count": [ 1, 6 ] },
        { "item": "afs_circuitry_3", "count": [ 0, 1 ] },
        { "item": "afs_circuitry_3", "count": [ 0, 1 ] },
        { "item": "afs_heat_2", "count": [ 0, 1 ] },
        { "item": "afs_energy_storage_2", "count": [ 1, 6 ] },
        { "item": "afs_energy_storage_4", "count": [ 0, 1 ] },
        { "item": "afs_material_1", "count": [ 12, 18 ] }
      ]
    }
  },
    {
    "type": "terrain",
    "id": "t_afs_gun_ship_engine_console_burnt",
    "name": "Iblis Interplanetary Drive Console",
    "looks_like": "t_console",
    "description": "The engine has been overloaded and burnt out.",
    "symbol": "0",
    "color": "green_red",
    "move_cost": 0,
    "coverage": 80,
    "examine_action": {
      "type": "effect_on_condition",
      "effect_on_conditions": [
        {
          "id": "EOC_AFS_GUNSHIP_DRIVE_OVERLOADED",
          "condition": { "u_query": "Do you want to overload the ship's drive?", "default": true },
          "effect": [
            { "u_message": "The engine is burnt out and worthless.", "popup": true }
          ]
        }
      ]
    },
    "flags": [ "NOITEM", "SEALED", "REDUCE_SCENT", "PERMEABLE" ],
    "bash": {
      "str_min": 70,
      "str_max": 400,
      "sound": "metal screeching!",
      "ter_set": "t_metal_floor",
      "sound_fail": "clang!",
      "items": [
        { "item": "afs_circuitry_1", "count": [ 1, 7 ] },
        { "item": "afs_circuitry_2", "count": [ 0, 2 ] },
        { "item": "afs_heat_2", "count": [ 0, 1 ] },
        { "item": "afs_material_1", "count": [ 8, 12 ] }
      ]
    }
  },
```